guides/user/canary/config/index.md

```diff
@@ -0,0 +1,169 @@
+but those [stages are defined separately](guides/user/canary/stage/).
+
+<!---
+TODO: figure out why this is done and how it's used
+--->
+Separately, you can [group individual metrics by Stackdriver
+resource types](#group_the_metrics).
+
+## Prerequisites
+
+By default, canary is not enabled for new applications. Several things need to
+happen before you see the __Canary__ tab in Deck:
+
+* The person setting up Spinnaker for you must [set up Canary](/setup/canary/).
+
+  This includes specifying whether all canary configs are available to all
+  applications or each application can have one or more configurations that are
+  visible to that application only.
+
+* In the Application config, activate the __Canary__ option.
+
+  Do this separately for all applications that will use automated canary
+  analysis.
+
+{%
+ include
+ figure
+ image_path="./enable_canary.png"
+%}
+
+## Create a canary configuration
+
+You can create as many of these as you like, and when you create a canary stage,
+you can select which configuration to use. Configurations you create within an
+application are available to all pipelines in that application, but your
+Spinnaker might be set up so that all configurations are available to all
+pipelines.
+
+1. Hover over the __Delivery__ tab, and select __Canary__.
+
+   ![Select __Canary__ from the __Delivery__ menu.](/guides/user/canary/config/delivery_menu_canary.png)
+1. Select __Add configuration__.
+
+1. Provide a __Name__ and __Description__.
+
+   This is the name shown in the stage config when you create a canary stage for your
+   pipeline.
+
+   ![Canary config declaration](/guides/user/canary/config/canary_config_create.png)
+
+The sections below describe how to specify the metrics and the scoring thresholds
+and weights.
+
+## Create metric groups and add metrics
+
+The metrics available depend on the telemetry provider you use. Spinnaker
+currently supports Stackdriver only.
+
+Metrics are evaluated even if they're not added to groups, but if you want to
+apply the weighting that determines the relative importance of different metrics,
+you need to [add them to groups](#create-metric-groups-and-add-metrics).
+
+1. In the __Metrics__ section, select __Add Metric__.
+
+1. Select the group to add this metric to.
+
+   Data is evaluated for all metrics, but metrics scores only affect the canary
+   evaluation for metrics that are grouped here.
+
+   Click __Add Group__ to create each group you'll use. For example, you might
+   create a group called  "cpu" and add a set of CPU-related metrics. Then,
+   when you configure a new metric here, you would select "cpu" for the
+   CPU-related metrics you add.
+
+1. Give the metric a name.
+
+1. Specify whether this metric fails when the value deviates too high or too low
+compared to the baseline.
+
+   Or select __either__, in which case it fails on deviation in either direction.
+
+1. Optionally, choose a [filter
```

Comment:
Let's talk about this a little more - filter templates aren't directly tied to Stackdriver filters, and are supported for prometheus.

Reply:
I have a whole article (currently empty) in which I'm going to explain about filter templates and how to use them. I'll probably just reference that from here. (Tho I've removed the Stackdriver-only mention below.)